### PR TITLE
tutorial formatting fixes

### PR DESCRIPTION
**Pull request by jpchen**
1. formatting fixes
2. change the pyro version to be consistent with __version__. i left the tag+commit code in there to uncomment when we approach 0.3 *TODO(jpchen) explain motivation*

### CODE REVIEW

File: tutorial/source/dmm.ipynb [hunk 1]
```diff
@@ -854,14 +856,14 @@
...
+    "version": 3
    },
    "file_extension": ".py",
    "mimetype": "text/x-python",
    "name": "python",
    "nbconvert_exporter": "python",
-   "pygments_lexer": "ipython2",
-   "version": "2.7.14"
+   "pygments_lexer": "ipython3",
+   "version": "3.6.2"
```
[the remark this answers is not in the document]
Comment (jpchen) [reply tutorial/source/dmm.ipynb]: arghh i forgot to revert this, i use py3 locally so any notebook i open will get changed..

**Comment by jpchen** [comment]
> Can you explain why you're removing the commit hash prefix in dev rather than in the 0.2-release branch

sorry this was wip, didnt expect you guys to jump so fast on this :D , but because there have been a lot of tutorial fixes merged to dev that i plan on deploying, but they build with the entire commit hash instead of 0.2.0. ill do a merge from dev to 0.2-release after all the tutorial fixes are in. on a side note it would be useful to have a clean 0.2.0 not have commit hashes appended to it

**Comment by jpchen** [comment]
ready for review/merge

**Comment by neerajprad** [comment]
Good to merge?

**Pull request merged**